pkg/ruler/manager.go

```diff
@@ -161,6 +163,22 @@ func (r *DefaultMultiTenantManager) syncRulesToManager(ctx context.Context, user
 	}
 }
 
+func ruleGroupIterationFunc(ctx context.Context, g *promRules.Group, evalTimestamp time.Time) {
+	logMessage := []interface{}{
+		"msg", "evaluating rule group",
+		"component", "ruler",
+		"rule_group", g.Name(),
+		"namespace", g.File(),
+		"num_rules", len(g.Rules()),
+		"alert_rule_count", len(g.AlertingRules()),
```

Comment:
Can we make the naming consistent with `num_rules` above? Maybe `num_alert_rules`.

Reply:
Done